### PR TITLE
Add Cloud Tasks snippets

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java [hunk 1]
```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright 2018 Google LLC
```
Comment (kurtisvg) [inline appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java]:
```suggestion
 * Copyright 2019 Google LLC
```

File: appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+      for (Queue queue : queues) {
+        Queue response = client.createQueue(parent, queue);
+        System.out.println(response);
+      }
+    }
+  }
+  // [END taskqueues_retrying_tasks]
+
+  // Run sample: mvn clean package exec:java -Dexec.mainClass="com.example.appengine.Snippets"
+  public static void main(String[] args) throws Exception {
```
Comment (kurtisvg) [inline appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java]: Can we remove this?

File: appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+import com.google.cloud.tasks.v2.RetryConfig;
+import com.google.cloud.tasks.v2.Task;
+import com.google.cloud.tasks.v2.TaskName;
+import com.google.cloud.tasks.v2.UpdateQueueRequest;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.Duration;
+import java.nio.charset.Charset;
+
+public class Snippets {
+  //  [START taskqueues_using_yaml]
```
Comment (kurtisvg) [inline appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java]: Should these follow the new format and be in different files?

File: appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java (hunk 1 above)
Comment (averikitsch) [reply appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java]: done.

File: appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java (hunk 2 above)
Comment (averikitsch) [reply appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java]: done.

File: appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java (hunk 3 above)
Comment (averikitsch) [reply appengine-java8/tasks/snippets/src/main/java/com/example/task/Snippets.java]: done.

File: appengine-java8/taskqueues-push/src/main/java/com/example/appengine/taskqueue/push/Enqueue.java [hunk 4]
```diff
@@ -43,9 +44,28 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 
     // Add the task to the default queue.
     // [START addQueue]
-    Queue queue = QueueFactory.getDefaultQueue();
-    queue.add(TaskOptions.Builder.withUrl("/worker").param("key", key));
-    // [END addQueue]
+    try (CloudTasksClient client = CloudTasksClient.create()) {
+      String projectId = "starter-akitsch";
```
Comment (kurtisvg) [inline appengine-java8/taskqueues-push/src/main/java/com/example/appengine/taskqueue/push/Enqueue.java]: 😲

Comment (averikitsch) [reply appengine-java8/taskqueues-push/src/main/java/com/example/appengine/taskqueue/push/Enqueue.java]: ?! I have no idea how that file got added to this PR?!

**Pull request merged**